src/main/java/edu/cornell/kfs/rass/batch/service/impl/RassUpdateServiceImpl.java

```diff
@@ -445,11 +455,44 @@ protected UserSession buildSessionForSystemUser() {
         maintenanceDocument.getDocumentHeader().setDocumentDescription(
                 buildDocumentDescription(newBo, maintenanceAction, objectDefinition));
         
+        try {
         maintenanceDocument = (MaintenanceDocument) documentService.routeDocument(
                 maintenanceDocument, annotation, null);
+        } catch(ValidationException ve) {
+            LOG.error("createAndRouteMaintenanceDocumentInternal, Error routing document # " + maintenanceDocument.getDocumentNumber() + " " + ve.getMessage(), ve);
+            String errorMessage = buildValidationErrorMessage((ValidationException) ve);
+            LOG.error("createAndRouteMaintenanceDocumentInternal: Could not route document - " + errorMessage);
+            throw new ValidationException(errorMessage);
+        }
         
         return maintenanceDocument;
     }
+    
+    public String buildValidationErrorMessage(ValidationException validationException) {
+        try {
+            Map<String, AutoPopulatingList<ErrorMessage>> errorMessages = GlobalVariables.getMessageMap().getErrorMessages();
+            return errorMessages.values().stream().flatMap(List::stream)
+                    .map(this::buildValidationErrorMessageForSingleError)
+                    .collect(Collectors.joining(KFSConstants.NEWLINE, KFSConstants.NEWLINE, KFSConstants.NEWLINE));
```

Comment:
Just to confirm, you want newline characters both before and after the generated error message string?

Reply:
Chad, yes, that would make it more clearly delimited but I could remove that if you think it is too many newlines.

Reply:
Thanks for clarifying, I just wanted to confirm that those newlines were intentional. I think this is find as-is, then.